components/service/glean/src/main/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngine.kt

```diff
@@ -76,19 +77,21 @@ internal open class TimingDistributionsStorageEngineImplementation(
             return
         }
 
+        val sampleInUnit = getAdjustedTime(timeUnit, sample)
+
         // Since the custom combiner closure captures this value, we need to just create a dummy
         // value here that won't be used by the combine function, and create a fresh
         // TimingDistributionData for each value that doesn't have an existing current value.
         val dummy = TimingDistributionData(category = metricData.category, name = metricData.name,
             timeUnit = timeUnit)
         super.recordMetric(metricData, dummy, null) { currentValue, newValue ->
```

Comment:
nit: since we're here, can we fix the `newValue` warning? If this isn't used, let's change it to `_`